test/shardingTable.test.js

```diff
@@ -1,42 +1,52 @@
 const {assert} = require('chai');
+const BN = require('bn.js');
 
 const ERC20Token = artifacts.require('ERC20Token');
 const Hub = artifacts.require('Hub');
 const Profile = artifacts.require('Profile');
 const ShardingTable = artifacts.require('ShardingTable');
+const ShardingTableStorage = artifacts.require('ShardingTableStorage');
+const IdentityStorage = artifacts.require('IdentityStorage');
+const ProfileStorage = artifacts.require('ProfileStorage');
+const ParametersStorage = artifacts.require('ParametersStorage');
+const bytes32 = require('bytes32');
```

Comment:
Do we need this?

Reply:
There is still more work to be done on tests, this should help with crafting values, tests with its usage will be pushed with next commit. Don't focus too much on tests yet.

Reply:
> There is still more work to be done on tests, this should help with crafting values, tests with its usage will be pushed with next commit. Don't focus too much on tests yet.

Np. I think that we should be consistent and use ethers.js for different kinds of type conversions etc. as we now do in other repos

Reply:
Good point